### DISCUSSION TITLE
Unaligned embeds

### DISCUSSION THREAD

**Discussion by prsh11** [Rapptz/discord.py#7254] [Q&A]
I am trying to create a help for my bot. But the fields seem to be unaligned

```py
embed = discord.Embed(title="Bluey's Command list", description="Thanks for using bluey. We will be adding new features bluey every now and then")
embed.set_thumbnail(url=self.client.user.avatar_url)
embed.add_field(name="🔒Admin", value=f"`{prefix}help Admin`")
embed.add_field(name="🥳 Levels", value=f"`{prefix}help Levels`")
embed.add_field(name="🕵️ Moderation", value=f"`{prefix}help Moderation`")
embed.add_field(name="💿 Music", value=f"`{prefix}help Music`")
embed.add_field(name="🐶 Animal", value=f"`{prefix}help Animal`")
embed.add_field(name="👀 Fun", value=f"`{prefix}help Fun`")
embed.add_field(name="Anime", value=f"`{prefix}help Anime`")
embed.add_field(name="Notifier", value=f"`{prefix}help Notifier`", inline=False)
```
Output:

![Embed](https://user-images.githubusercontent.com/67903361/126175861-1e3f40bc-ae1c-4938-808f-5012627aac47.png)

**Comment by Rapptz** [comment]
discord.py does not control embeds -- their rendering is entirely client sided and client specific. Generally the workaround here is to add another field with a name and value of `'\u200b'` but this is hacky. Fundamentally bots have no fine tuned control over the rendering.

**Comment by Jonah987654321** [comment]
Otherwise, if you find, that this looks better, make all fields not inline, by adding 
```
inline = False 
``` 
when you use `add_field`
